composer.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "moxie-leean/LeeanPlugin",
+  "description": "Barebones modular WordPress plugin",
+  "type": "wordpress-plugin",
+  "require": {
+    "composer/installers": "^1.0",
+    "moxie-lean/wp-endpoint": "*"
+  },
+  "require-dev": {
+    "squizlabs/php_codesniffer": "2.*",
+    "wp-coding-standards/wpcs": "0.9.0"
+  },
+  "authors": [
+    {
+      "name": "Moxie",
+      "email": "devs@getmoxied.net"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "LeanWoocommerceApi\\": "src/"
```

Comment:
This needs to specify what namespace starts on `src/ ` I think in this case is `Lean\\Woocomerce\\Api\\`